components/calendar/src/ethiopic.rs

```diff
@@ -45,17 +45,10 @@ use tinystr::tinystr;
 
 // The Ethiopic Calendar
 #[derive(Copy, Clone, Debug, Hash, Default, Eq, PartialEq)]
-#[allow(clippy::exhaustive_structs)] // this type is stable
-pub struct Ethiopic;
+pub struct Ethiopic(pub(crate) bool);
```

Comment:
Note: This is why I had earlier advocated for making the calendars be `#[non_exhaustive]`... in case we wanted to do something like add a private field to them

Reply:
I think this is a special weird case where we did have intent to do this from the start but the work was somewhat WIP. I don't see this occurring again except for other calendars where we _know_ they'll have something in there